code/modules/mob/living/carbon/human/species_types/lizardpeople.dm

```diff
@@ -25,6 +25,14 @@
 	deathsound = 'sound/voice/lizard/deathsound.ogg'
 	wings_icon = "Dragon"
 	species_language_holder = /datum/language_holder/lizard
+	// lizards are coldblooded
+	// and can stand aa higher tempurature range by a few points
+	bodytemp_heat_damage_limit = (BODYTEMP_HEAT_DAMAGE_LIMIT + 10)
+	bodytemp_cold_damage_limit = (BODYTEMP_COLD_DAMAGE_LIMIT - 10)
+
+// Lizards are cold blooded and do not stabilize body tempurature naturally
+/datum/species/lizard/natural_bodytemperature_stabilization(mob/living/carbon/human/H)
+	return 0

```

Comment:
so do lizardpeople now just straight up die if they don't chug a temperature-altering drink after getting set on fire (even if they extinguish themselves) or cryostung once?

Reply:
Interesting, would that not be the case to real cold blooded?
What would be reasonable... drop down to 5 under heat damage over time? or one over head damage?
But this is only the natural stabilization. Lizards still normalize to the room once they are out of the fire and boiling hot air. the stations 20c will do that for any lizard very fast. (species.dm 1672) they get all the environment change but none of the natural change.